File: src/ReparationBundle/Entity/Reparation.php
<?php

namespace ReparationBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reparation
 *
 * @ORM\Table(name="reparation")
 * @ORM\Entity(repositoryClass="ReparationBundle\Repository\ReparationRepository")
 */
class Reparation
{


    /**
     * @ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     * @ORM\JoinColumn(name="userId", referencedColumnName="id")
     */
    private $user;

    /**
     * @return mixed
     */
    public function getReparateur()
    {
        return $this->user;
    }

    /**
     * @param mixed $reparateur
     */
    public function setReparateur($reparateur)
    {
        $this->user = $reparateur;
    }

    /**
     * @ORM\ManyToOne(targetEntity="Piecesdefectueuses")
     * @ORM\JoinColumn(name="Piecesdefectueuses_id", referencedColumnName="id")
     */
    private $piecedefectueuse;


    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @return mixed
     */
    public function getUser()
    {
        return $this->user;
    }

    /**
     * @param mixed $user
     */
    public function setUser($user)
    {
        $this->user = $user;
    }

    /**
     * @return \DateTime
     */
    public function getDateFin()
    {
        return $this->dateFin;
    }

    /**
     * @param \DateTime $dateFin
     */
    public function setDateFin($dateFin)
    {
        $this->dateFin = $dateFin;
    }
    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateFin", type="datetime")
     */
    private $dateFin;

    /**
     * @var float
     *
     * @ORM\Column(name="prixRep", type="float")
     */
    private $prixRep;
    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateDebut", type="datetime")
     */
    private $dateDebut;

    /**
     * @return \DateTime
     */
    public function getDateDebut()
    {
        return $this->dateDebut;
    }

    /**
     * @param \DateTime $dateDebut
     */
    public function setDateDebut($dateDebut)
    {
        $this->dateDebut = $dateDebut;
    }





    /**
     * @var string
     *
     * @ORM\Column(name="description", type="text")
     */
    private $description;

    /**
     * @return mixed
     */
    public function getPiecedefectueuse()
    {
        return $this->piecedefectueuse;
    }

    /**
     * @param mixed $piecedefectueuse
     */
    public function setPiecedefectueuse($piecedefectueuse)
    {
        $this->piecedefectueuse = $piecedefectueuse;
    }


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set prixRep
     *
     * @param float $prixRep
     *
     * @return Reparation
     */
    public function setPrixRep($prixRep)
    {
        $this->prixRep = $prixRep;

        return $this;
    }

    /**
     * Get prixRep
     *
     * @return float
     */
    public function getPrixRep()
    {
        return $this->prixRep;
    }






    /**
     * Set description
     *
     * @param string $description
     *
     * @return Reparation
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }



}

File: web/java/huntkingdom/src/Entities/Reparation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;

/**
 *
 * @author asus_pc
 */
public class Reparation {
    private int id;
    private Date dateFin;
    private Date dateDebut;

    public Date getDateDebut() {
        return dateDebut;
    }

    public void setDateDebut(Date dateDebut) {
        this.dateDebut = dateDebut;
    }
    private Double prixRep;
    private String description;
    private int userId;
    private int Piecesdefectueuses_id;

    public Reparation() {
    }

    public Reparation(int id,Date dateDebut, Date dateFin, Double prixRep, String description, int userId, int Piecesdefectueuses_id) {
        this.id = id;
        this.dateDebut = dateDebut;
        this.dateFin = dateFin;
        this.prixRep = prixRep;
        this.description = description;
        this.userId = userId;
        this.Piecesdefectueuses_id = Piecesdefectueuses_id;
    }
       public Reparation( Date dateDebut,Date dateFin, Double prixRep, String description, int userId, int Piecesdefectueuses_id) {
        this.dateDebut = dateDebut;
        this.dateFin = dateFin;
        this.prixRep = prixRep;
        this.description = description;
        this.userId = userId;
        this.Piecesdefectueuses_id = Piecesdefectueuses_id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Date getDateFin() {
        return dateFin;
    }

    public void setDateFin(Date dateFin) {
        this.dateFin = dateFin;
    }

    public Double getPrixRep() {
        return prixRep;
    }

    public void setPrixRep(Double prixRep) {
        this.prixRep = prixRep;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getPiecesdefectueuses_id() {
        return Piecesdefectueuses_id;
    }

    public void setPiecesdefectueuses_id(int Piecesdefectueuses_id) {
        this.Piecesdefectueuses_id = Piecesdefectueuses_id;
    }
    
}
